examples/AspNetCore/Startup.cs

```diff
@@ -52,35 +52,39 @@ public void ConfigureServices(IServiceCollection services)
                 }
             });
 
-            // Switch between Zipkin/Jaeger by commenting out one of the following.
-
-            /*
-            services.AddOpenTelemetry((builder) => builder
-                .AddAspNetCoreInstrumentation()
-                .AddHttpInstrumentation()
-                .UseJaegerActivityExporter(o =>
-                {
-                    o.ServiceName = this.Configuration.GetValue<string>("Jaeger:ServiceName");
-                    o.AgentHost = this.Configuration.GetValue<string>("Jaeger:Host");
-                    o.AgentPort = this.Configuration.GetValue<int>("Jaeger:Port");
-                }));
-            */
-
-            /*
-            services.AddOpenTelemetry((builder) => builder
-                .AddAspNetCoreInstrumentation()
-                .AddHttpInstrumentation()
-                .UseZipkinExporter(o =>
-                {
-                    o.ServiceName = this.Configuration.GetValue<string>("Zipkin:ServiceName");
-                    o.Endpoint = new Uri(this.Configuration.GetValue<string>("Zipkin:Endpoint"));
-                }));
-            */
-
-            services.AddOpenTelemetry((builder) => builder
-                .AddAspNetCoreInstrumentation()
-                .AddHttpClientInstrumentation()
-                .UseConsoleExporter());
+            // To choose which exporter to use, run this example with the UseExporter option (defaults to the ConsoleExporter):
```

Comment:
lets use appsettings.json for now - all other settings are configured there, so why not the choice of exporter as well.